File: clients/javascript/src/models/topicConfigureOut.ts
// this file is @generated





export interface TopicConfigureOut {
    partitions: number;
}

export const TopicConfigureOutSerializer = {
    _fromJsonObject(object: any): TopicConfigureOut {
        return {
            partitions: object['partitions'],
            };
    },

    _toJsonObject(self: TopicConfigureOut): any {
        return {
            'partitions': self.partitions,
            };
    }
}

File: clients/rust/src/models/topic_configure_out.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[non_exhaustive]
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct TopicConfigureOut {
    pub partitions: u16,
}

impl TopicConfigureOut {
    pub fn new(partitions: u16) -> Self {
        Self { partitions }
    }
}
